thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/datasource/sql/SqlUtils.java

```diff
@@ -581,6 +581,15 @@ private static String timeFormatToMySQLFormat(String timeFormat) {
     }
   }
 
+  private static String timeFormatToVerticaFormat(String timeFormat) {
+    if(timeFormat.contains("mm")){
+        return timeFormat.replace("mm","mi");
```

Comment:
Can we perform a case-insensitive replace?
https://stackoverflow.com/questions/5054995/how-to-replace-case-insensitive-literal-substrings-in-java